events/2021.09.Online/TD/TDs/NHK/displayEmulator.td.jsonld

```diff
@@ -17,10 +17,10 @@
         "power": {
             "title": "power",
             "@type": "saref:OnOffState",
-            "type": "string",
+            "type": "boolean",
```

Comment:
Property `power` returns `{"status": true}` or `{"status": false}`.
So I think correct schema is following:
```json
"type": "object",
  "properties": {
    "status": {
      "type": "boolean"
    }
  }
}
```